demos/simple-menu.html

```diff
@@ -60,14 +60,15 @@
 
       .mdc-theme--dark {
         background-color: #333;
+        color: #ddd;
```

Comment:
Legible text is nice feature 👍 

Updating L43 from `margin-top: 16px;` to `padding: 16px 0;` would remove the white gap between the hero and the demo when dark mode is activated. Makes it look just a tad cleaner!

Reply:
Good call! Done.